### ISSUE TITLE
catalog-backend-module-msgraph still recommends removed runPeriodically

### ISSUE THREAD

**Issue by pjungermann** [backstage/backstage#10141]
## Expected Behavior

Documentation for catalog-backend-module-msgraph allows users to initially configure the provider or processor by following it.

## Actual Behavior

at the README.md, the plugin recommends to use

```typescript
// Trigger a read every 5 minutes
useHotCleanup(
  module,
  runPeriodically(() => msGraphOrgEntityProvider.read(), 5 * 60 * 1000),
);
```

## Your Environment

- runPeriodically was deprecated in release 0.69.0
- runPeriodically was removed in release 0.71.0

**Comment by freben** [comment]
Good catch, the providers should all be given this treatment https://github.com/backstage/backstage/pull/10018

**Comment by pjungermann** [comment]
yes, makes sense and thanks for the example PR. After looking into backend-tasks a bit as a mentioned replacement for the removed method I thought about the same, adding this capability to the processor/provider.

**Comment by pjungermann** [comment]
as a workaround, I took the "copy runPeriodically implementation" option for now with a TODO to revisit later when this was resolved. I will have another look at another day.

**Comment by pjungermann** [comment]
Closed by #10227
